scale-out-relay/src/main/scala/com/normation/plugins/scaleoutrelay/DataTypes.scala

```diff
@@ -82,10 +82,10 @@ class ScaleOutRelayAgentSpecificGeneration(pluginInfo: PluginStatus) extends Age
   import com.normation.rudder.services.policies.write.BuildBundleSequence.{ BundleSequenceVariables, InputFile, TechniqueBundles }
   override def getBundleVariables(
       systemInputs: List[InputFile]
-    , sytemBundles: List[TechniqueBundles]
```

Comment:
@fanf is this the correct name? the same typo is found multiple time so I am not sure if it is really a typo or just a bad named thing

Reply:
It appears to be a typo we can't fix as it is part of the plugin's API

Reply:
up